config/action.d/sendmail-geoip-lines.conf

```diff
@@ -0,0 +1,48 @@
+# Fail2Ban configuration file
+#
+# Author: Viktor Szépe
+#
+#
+
+[INCLUDES]
+
+before = sendmail-common.conf
+
+[Definition]
+
+# Option:  actionban
+# Notes.:  Command executed when banning an IP. Take care that the
+#          command is executed with Fail2Ban user rights.
+#          You need to install geoiplookup and the GeoLite or GeoIP databases.
+#          (geoip-bin and geoip-database-contrib in Debian)
```

Comment:
I shouldn't forget to add those (probably geoip-database to stay within main) to Debian's package Suggests: